src/app/components/SheriffListCard/SheriffListCard.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import * as moment from 'moment';
+import { Sheriff } from '../../api/index';
+import { 
+    ListGroupItem
+} from 'react-bootstrap';
+import ScheduleSummary from '../../containers/ScheduleSummary';
+import './SheriffListCard.css';
+
+export interface SheriffListCardProps {
+    onClick?: () => void;
+    sheriff: Sheriff;
+    showScheduleSummary?: boolean;
+}
+export default class SheriffListCard extends React.PureComponent<SheriffListCardProps, {}> {
+    render() {
+        const { sheriff, showScheduleSummary = false } = this.props;
+        const { firstName, lastName, badgeNumber, id } = sheriff;
+
+        return (
+            <div className="sheriff-list-card">
```

Comment:
Do we need this outer div?  I see (from the css above) that your using it for padding mostly.  I think we could probably replace the padding with the equivalent margin perhaps?  Just a thought.